src/sentry/integrations/vsts/notify_action.py

```diff
@@ -34,5 +35,34 @@ def __init__(self, *args, **kwargs):
     def render_label(self):
         return self.label.format(name=self.get_integration_name())
 
+    def build_description(self, event, installation):
+        rule_url = u"/organizations/{}/alerts/rules/{}/{}/".format(
+            self.project.organization.slug, self.project.slug, self.rule.id
+        )
+        footer = u"\nThis ticket was automatically created by Sentry via [{}]({})".format(
+            self.rule.label, absolute_uri(rule_url),
+        )
+        return installation.get_group_description(event.group, event) + footer
+
     def after(self, event, state):
```

Comment:
@ceorourke I wish we could move `after` to the base class I suggested above `TicketEventAction` but we'll probably need an abstraction to map the response to the `key` we use on line 61.

Reply:
@scefali I was thinking about this too but there are still some things to add to `after` like checking if a ticket has been created already and creating the grouplink (I think? haven't totally figured that part out yet) that I don't _think_ will diverge but who knows ... anyway I figured it made more sense to do that later when I have a more complete picture.

Reply:
sounds good to me!